### PR TITLE
Links now change color across site

### PR DESCRIPTION
**Pull request by heathermoses**
- [ x] No discrepancies across browsers. Eg. chrome vs safari
- [x ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [x ] CSS (https://jigsaw.w3.org/css-validator/)
- [x ] Google sign-in is operational
- [ x] Accessibility functions
- [x ] Pages can scale without distorting page
- [ x] No dead links
- [ x] Navbar is consistent across the site
- [ x] Pages are screen-reader accessible
- [ x] Contrast meets standards for accessibility
- [ x] Pages are keyboard accessible
- [ x] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,5 +15,8 @@
   "bugs": {
     "url": "https://github.com/all-rit/AccessibilityLab1/issues"
   },
-  "homepage": "https://github.com/all-rit/AccessibilityLab1#readme"
+  "homepage": "https://github.com/all-rit/AccessibilityLab1#readme",
```
Comment (sk6786) [inline package.json:18]: Is there a reason for this change, or was it committed by accident

File: server/package.json [hunk 2]
```diff
@@ -14,13 +14,14 @@
     "cookie-parser": "latest",
     "cookie-session": "latest",
     "cors": "latest",
```
Comment (sk6786) [inline server/package.json:16]: Same with this file

**Review by sk6786 — COMMENTED** [review]
Looks great, and works as expected. Just look over the package-lock and package-json files. I see there was one file deleted. If they were committed by mistake just revert them.

File: package.json (hunk 1 above)
Comment (heathermoses) [reply package.json:18]: I'm not sure how that got changed, but it definitely doesn't need to be there. Good catch

File: server/package.json (hunk 2 above)
Comment (heathermoses) [reply server/package.json]: Good catch, should be fixed now

**Pull request merged**